serverless/pages/custom-roles.mdx

```diff
@@ -0,0 +1,105 @@
+{/* TO-DO: This screenshot needs to be refreshed and automated. */}
+
+Refer to [cluster privileges](((ref))/security-privileges.html#privileges-list-cluster) for a complete description of available options.
+
+{/* Derived from https://www.elastic.co/guide/en/kibana/current/kibana-role-management.html#adding_cluster_privileges */}
+
+## ((es)) index privileges
+
+Each role can grant access to multiple data indices, and each index can have a different set of privileges.
+Typically, you will grant the `read` and `view_index_metadata` privileges to each index that you expect your users to work with.
+For example, grant access to indices that match an `acme-marketing-*` pattern:
+
+![Create a custom role and define ((es)) index privileges](../images/custom-roles-index-privileges.png)
+{/* TO-DO: This screenshot needs to be refreshed and automated. */}
+
+Refer to [index privileges](((ref))/security-privileges.html#privileges-list-indices) for a complete description of available options.
+
+Document-level and field-level security affords you even more granularity when it comes to granting access to your data.
+With document-level security (DLS), you can write an ((es)) query to describe which documents this role grants access to.
+With field-level security (FLS), you can instruct ((es)) to grant or deny access to specific fields within each document.
+{/* Derived from https://www.elastic.co/guide/en/kibana/current/kibana-role-management.html#adding_cluster_privileges */}
+
+## ((kib)) privileges
+
+When you create a custom role, click **Add Kibana privilege** to grant access to specific features.
+The features that are available vary depending on the project type.
+For example, in ((es-serverless)):
+
+![Create a custom role and define ((kib)) privileges](../images/custom-roles-kibana-privileges.png)
+{/* TO-DO: This screenshot needs to be refreshed and automated. */}
+
+Open the **Spaces** selection control to specify whether to grant the role access to all spaces or one or more individual spaces.
+When using the **Customize by feature** option, you can choose either **All**, **Read** or **None** for access to each feature.
+
+All
+: Grants full read-write access.
+
+Read
+: Grants read-only access.
+
+None
+: Does not grant any access.
+
+Some features have finer access control and you can optionally enable sub-feature privileges.
+
+<DocCallOut title="New features">
+As new features are added to ((serverless-full)), roles that use the custom option do not automatically get access to the new features. You must manually update the roles.
+</DocCallOut>
+
+{/* Derived from https://www.elastic.co/guide/en/kibana/current/kibana-role-management.html#adding_kibana_privileges and https://www.elastic.co/guide/en/kibana/current/kibana-privileges.html */}
+
+After your roles are set up, the next step to securing access is to assign roles to your users.
+Learn more in <DocLink slug="/serverless/general/assign-user-roles" /> 
```

Comment:
Can reference the Assign roles link button at the top of the page. This will take the user to the cloud org members page.

Reply:
Will the `assign-user-roles` page get updated as well?

Reply:
Yes, I'll handle that in a separate PR